src/types/BscType.ts

```diff
@@ -10,6 +10,7 @@ export abstract class BscType {
     public readonly memberTable: SymbolTable;
     protected __identifier: string;
     protected hasAddedBuiltInInterfaces = false;
+    public isBuiltIn = false;
```

Comment:
Added a flag at the type level to basically say "this is a type we don't need to mess with"